kitsune/sumo/management/commands/extract.py

```diff
@@ -0,0 +1,63 @@
+from django.core.management.base import BaseCommand
+from babel.messages.frontend import CommandLineInterface
+
+
+class Command(BaseCommand):
+    """
+    Allows you to extract strings for localization.
+    Extract creates two .pot files - django.pot and djangojs.pot - that contain
+    messages to merge into the existing l10n files.
+
+    """
+
+    help = "Extracts localizable strings from the codebase."
+
+    def handle(self, *args, **options):
+        CommandLineInterface().run(
+            [
+                "pybabel",
+                "extract",
+                "-F",
+                "babel.cfg",
+                "-o",
+                "locale/templates/LC_MESSAGES/django.pot",
+                "-k",
+                "_lazy",
+                "-k",
+                "pgettext_lazy",
```

Comment:
Since `pgettext_lazy` is special in that it accepts 2 arguments, with the first being the context for the translation, we have to inform the extract command about that special case. I discovered and tested that you do that as suggested below.
```suggestion
                "pgettext_lazy:1c,2",
```

Reply:
Great catch AND solution. woot!